.github/workflows/_e2e_tests.yaml

```
@@ -134,6 +134,25 @@ jobs:
         with:
           password: ${{ secrets.PULP_PASSWORD }}
 
+      - name: split image and tag
+        id: split
+        env:
+          KONG: ${{ inputs.kong-image }}
+          CONTROLLER: ${{ inputs.controller-image }}
+        run: |
+          if [ "${{ inputs.kong-image }}" != "" ]; then
+            export kong_image=$(echo ${{ inputs.kong-image }} | awk '{split($0,a,":"); print a[1]}')
+            export kong_tag=$(echo ${{ inputs.kong-image }} | awk '{split($0,a,":"); print a[2]}')
```

Those could simple be (note the `KONG` env added in line 140)

```suggestion
            export kong_image=$(echo ${KONG%%:*})
            export kong_tag=$(echo ${KONG##*:})
```


```
$ export IMG=kong/kubernetes-ingress-controller:2.12.0
$ echo ${IMG%%:*}
kubernetes-ingress-controller
$ echo ${IMG##*:}
2.12.0
```

using [parameter expansion](https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html#:%7E:text=the%20last%20element.-,%24%7Bparameter%23word%7D,the%20array%20in%20turn%2C%20and%20the%20expansion%20is%20the%20resultant%20list.,-%24%7Bparameter/)

---

And instead of `exporting` we could (should?) use the `GITHUB_OUTPUT`

```
            echo "kong_image=$(echo ${KONG%%:*})" >> $GITHUB_OUTPUT
```

Simplifies 2 lines into 1.

---

Could, but we don't need pure bash here, and I can read what the awk split on char/print split array index is doing more easily than the bash runes.